dockstore-webservice/src/main/java/io/dockstore/webservice/jdbi/WorkflowDAO.java

```diff
@@ -175,6 +178,73 @@ public WorkflowDAO(SessionFactory factory) {
         return filteredWorkflows.size() == 1 ? Optional.of(filteredWorkflows.get(0)) : Optional.empty();
     }
 
+    @SuppressWarnings({"checkstyle:ParameterNumber"})
+    public List<Workflow> filterTrsToolsGet(String descriptorType, String registry, String organization, String name, String toolname,
+            String description, String author, Boolean checker) {
+
+        SourceControlConverter converter = new SourceControlConverter();
+        CriteriaBuilder cb = currentSession().getCriteriaBuilder();
+        CriteriaQuery<Workflow> q = cb.createQuery(Workflow.class);
+        Root<Workflow> entryRoot = q.from(Workflow.class);
+
+        List<Predicate> predicates = new ArrayList<>();
+        predicates.add(cb.isTrue(entryRoot.get("isPublished")));
```

Comment:
I don't think you need this nor the previous line.

Reply:
To potentially clarify, I think you need this in general (the query should only return published workflows) but I think this is currently unused which means that this might be returning unpublished content which is not good.

Reply:
As I read it, the next line does do the isPublished, so I think these lines are redundant (and not even used)